examples/basic.cpp

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <beman/dump/dump.hpp>
+
+int main()
+{
+    beman::dump::dump(42, "Hello World!", 3.14, nullptr);
```

Comment:
From [cppreference](https://en.cppreference.com/w/cpp/language/main_function):

> The body of the main function does not need to contain the [return statement](https://en.cppreference.com/w/cpp/language/return): if control reaches the end of main without encountering a return statement, the effect is that of executing return 0;.

Reply:
I agree with you, but is this pattern that we would like to promote in Beman examples? :D 

* exemplar and optional26 do not follow it
* https://github.com/beman-project/execution26/blob/main/examples/ seems to use the pattern

Reply:
> is this pattern that we would like to promote in Beman examples?

Omitting needless code is a good practice. I created https://github.com/beman-project/exemplar/issues/33 to update beman.exemplar's examples.

Reply:
would you like to add a recommendation to the standard? :D

Reply:
I'm thinking this stylistic piece probably isn't important enough to be enshrined in the standard.

Reply:
Ok here's an opinion.  I personally like to make the return explicit even though I know the compiler will add it -- maybe it's just habit because all non-trivial applications tend to have multiple exit points -- some with errors. 

As for adding it to the standard for beman, eh -- it seems too trivial to mention. We should keep the guidance to more important things I think.